src/Locations.js

```diff
@@ -202,55 +206,71 @@ export default class Locations extends Component {
-                                                isDropDisabled={this.state[location].length > 0}>
-                                                {this.state[location].map((item, index) => (
-                                                    <Draggable
-                                                        key={item}
-                                                        draggableId={item}
-                                                        index={index}>
-                                                        {(provided, snapshot) => (
-                                                            <div
-                                                                ref={provided.innerRef}
-                                                                {...provided.draggableProps}
-                                                                {...provided.dragHandleProps}
-                                                                style={getItemStyle(
-                                                                    snapshot.isDragging,
-                                                                    provided.draggableProps.style
-                                                                )}>
-                                                                {item.split('-').join(' ')}
-                                                            </div>
-                                                        )}    
-                                                    </Draggable>
-                                                ))}
-                                            </DroppableLocation>
-                                        ))}
-                                        </div>
+                                      <h1>{locationGroup}</h1>
+                                      <Tabs>
+                                        <TabList>
+                                          {Object.keys(LOCATIONS[locationGroup]).map((locationName) =>(
+                                            <Tab key={locationName.split(' ').join('-')}>{locationName}</Tab>
+                                          ))}
+                                        </TabList>
+                                          {Object.entries(LOCATIONS[locationGroup]).map((locationName) => (
+                                            <TabPanel>
+                                                  <div style={{
+                                                      display: 'flex',
+                                                      flexWrap: 'wrap',
+                                                      justifyContent: 'space-around',
+                                                      maxHeight: '75vh',
+                                                      overflowY: 'auto'
+                                                  }}>
+                                                  {Object.values(locationName[1]).map((location) => (
```

Comment:
Wouldn't this just be a regular array, making `Object.values` unnecessary?

Reply:
You are absolutely correct. That'd be an issue where I get into a pattern and then stop thinking outside of the box. Thank you